test/unit/expressions.test.js

```diff
@@ -1,40 +0,0 @@
-const { expect } = require('chai');
-const { expressions, BASE_SCC_URL } = require('../../expressions.js');
-
-
-const findMatching = (url) => {
-  const parsedURL = new URL(url);
-  const pathname = parsedURL.pathname;
-  for (let pathType of expressions) {
-      const match = pathname.match(pathType.expr);
-      if (match) { return pathType.name; }
-  }
-}
-
-describe('expressions', () => {
```

Comment:
Were these tests no longer useful?

Reply:
these tests were for a file of expressions that were not being used anymore